great-idea/css/index.css

```diff
@@ -63,4 +58,177 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 876px;
+	margin: 0 auto;
+}
+
+.navigation {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	width: 876px;
```

Comment:
in this case where the parent div (you set it as your body tag) already has the width hard coded to 876, instead of hard coding it to other container elements, it would be safer to use with 100%. That way if the container divs width ever change, all the elements below it will follow and you won't have to manually change them all.